src/loader.ts

```diff
@@ -385,6 +389,40 @@ interface ModuleDefinitionArguments extends Array<any> {
 		pendingCacheInsert = {};
 	}
 
+	function loadNodeModule(moduleId: string, parent?: Module): any {
```

Comment:
Why did we move this?  In the future, when we have a builder that can optimise based on has flags, this won't now be elided.